### PR TITLE
Update AppLifecycle Environment Variables.md

### PR DESCRIPTION
**Pull request by andreww-msft**

### CODE REVIEW

**Comment by riverar** [comment]
Hm. This is very confusing and will introduce unexpected behavior. What's the value in doing any of this? Does anyone want this behavior?

File: specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md [hunk 1]
```diff
@@ -113,8 +113,9 @@ Notes:
   if an app that has done so is now uninstalled.
 - We won’t track changes that are made outside the API; and we won’t track
   uninstall of unpackaged apps.
-- When a packaged app is uninstalled, any variable it created will be deleted,
-  regardless of any other changes made to that variable during its lifetime.
+- When a packaged app is uninstalled, a variable it created will be deleted,
+  unless another packaged app also wrote to that variable (and that other app
+  has not yet been uninstalled) - in which case the previous value is restored.
```
[the remark this answers is not in the document]
Comment (andreww-msft) [reply specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md:118]: Actually, 
Point 3 is covered in example 3.
Point 4 is covered in example 6.
Point 5 is covered in example 6.

**Comment by andreww-msft** [comment]
> Hm. This is very confusing and will introduce unexpected behavior. What's the value in doing any of this? Does anyone want this behavior?

Hi Riverar, can you elaborate on what aspect is confusing? The value is that we've had multiple customer requests over the years for supporting environment variables in a consistent set of APIs, so yes folks have asked for this behavior.

File: specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhoehna) [reply specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md:118]: Sorry my bad.  I must have read the examples wrong.

**Comment by bpulliam** [comment]
Closing stale pull requests over a year old

**Pull request closed without merging**